src/http/content.cr

```diff
@@ -192,7 +192,7 @@ module HTTP
     end
 
     private def read_chunk_size
-      line = @io.read_line(HTTP::MAX_HEADER_SIZE, chomp: true)
+      line = @io.read_line(HTTP::MAX_HEADERS_SIZE, chomp: true)
```

Comment:
Do you think we can make `HTTP::Content` hold a `@max_headers_size` and set it in the constructor? Then it will take the same value configured in `HTTP::Server`.

Reply:
Sure, that should work :+1:

Reply:
Only `HTTP::ChunkedContent` needs this.

Reply:
Ah, yes, sorry, I meant to say `HTTP::ChunkedContent`